packages/server/package.json

```diff
@@ -94,7 +94,7 @@
 	"dependencies": {
 		"@hono/node-server": "^1.13.7",
 		"@hono/valibot-validator": "^0.5.1",
-		"hono": "^4.6.13",
+		"hono": "^4.6.14",
```

Comment:
Another idea to consider is to setup Renovate CI, so we can worry less about those changes and missing something.